.changes/v2.22.0/597-improvements.md

```diff
@@ -0,0 +1 @@
+* Add `VdcGroup.ForceDelete` function to optionally force VDC Group removal [GH-597]
```

Comment:
It would be beneficial for the readers to have an explanation when it is useful. I mean:

```suggestion
* Add `VdcGroup.ForceDelete` function to optionally force VDC Group removal (useful when ...) [GH-597]
```

Reply:
Addressed